src/classes/HouseholdName.cls

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
I believe this is a service class as opposed to a domain class, so a class name such as `HouseholdNameService` or `HouseholdName_SVC` would be more appropriate.

Reply:
Yeah, this might call for a renaming of HH_HouseholdNaming - I'm going to look at that class and this one and see if a few renames and moving a couple methods would have these all make a bit more sense together.

Reply:
Renamed HH_HouseholdNaming to HouseholdNamingService, which clarifies its role, especially in relation to the HouseholdName value object.  Let me know what you think!